doc/decisions/0009-telephone-and-email-placeholder-transforms.md

```diff
@@ -0,0 +1,73 @@
+# 9. Telephone and email placeholder transforms
+
+## Context
+
+When rendering telephone numbers and email addresses, we need to be able to wrap them with specific markup. This markup should not be included in schema text as it:
+- binds the schema to the output format 
+- creates translation issues
+
+## Proposal
+
+Introduce two new placeholder transforms that abstract the markup from the text.  
+
+### Telephone number link transform
+
+The `telephone_number_link` transform will allow runner to wrap the number with a `tel` href.
+
+```json
+{
+  "text": "Ring us on {telephone_number}",
+  "placeholders": [
+    {
+      "placeholder": "telephone_number",
+      "transforms": [
+        {
+          "transform": "telephone_number_link",
+          "arguments": {
+            "telephone_number": "0300 1234 931"
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+### Email link transform
+
+The `email_link` transform will allow runner to wrap the email address with a `mailto` href and a customisable email subject. 
+
+The `email_subject` and `email_subject_append` variables are optional, if provided they will be used to set the email subject line.
+
+```json
+{
+  "text": "Email us at {email_address}",
+  "placeholders": [
+    {
+      "placeholder": "email_address",
+      "transforms": [
+        {
+          "transform": "email_link",
+          "arguments": {
+            "email_address": "surveys@ons.gov.uk",
+            "email_subject": "Change of details reference ",
```

Comment:
Makes sense for the time being. Do we want to allow defining the text for the href also, or do we want to enforce it will be same as tel/email itself? Either way, translation would be an issue until we do like recursive placeholders.

Reply:
The current requirement is that the link text is always the phone number or email address, so didn't made it customisable. There is no translation issue with this as the phone number or email address don't need to be translated.

Reply:
I think the best practice is for the link text (telephone / email address) to be the value of the target so this is fine.